src/app/dynamic-forms/service/print/print.service.ts

```diff
@@ -325,7 +325,15 @@ export class PrintService {
     if (multiFields?.length) {
 
       let tableColumns = Object.keys(multiFields[0]);
-
+      multiFields.forEach(field => {
```

Comment:
Might be a merge issue again for this print.service.ts, seems like it's missing a recent bug fix